lasting_effect.cpp

```diff
@@ -536,7 +536,7 @@ const char* LastingEffects::getName(LastingEffect type) {
     case LastingEffect::MAGIC_VULNERABILITY: return "magic vulnerability";
     case LastingEffect::MELEE_VULNERABILITY: return "melee vulnerability";
     case LastingEffect::RANGED_VULNERABILITY: return "ranged vulnerability";
-    case LastingEffect::DARKNESS_SOURCE: return "source of darkness";
+    case LastingEffect::DARKNESS_SOURCE: return "night"; //This adjective fits on-screen properly.
```

Comment:
"source of darkness" should fit fine on screen, can you tell me where it doesn't?